### PR TITLE
Clean up English og.task translations.

### PR DESCRIPTION
**Pull request by lukasgraf**
Clean up English `og.task` translations.

For overall comments on the process, please see PR #6806

Jira: https://4teamwork.atlassian.net/browse/CA-883

### CODE REVIEW

File: opengever/task/locales/en/LC_MESSAGES/opengever.task.po [hunk 1]
```diff
@@ -397,65 +398,65 @@ msgstr "Duration in h"
...
 msgid "help_task_type"
-msgstr "help_task_type"
+msgstr "Choose task type"
 
 #. German translation: Geben Sie eine detaillierte Arbeitsanweisung oder einen Kommentar ein
 #: ./opengever/task/browser/delegate/metadata.py
 #: ./opengever/task/task.py
 msgid "help_text"
-msgstr "help_text"
+msgstr ""
```
Comment (njohner) [inline opengever/task/locales/en/LC_MESSAGES/opengever.task.po:443]: ?

File: opengever/task/locales/en/LC_MESSAGES/opengever.task.po [hunk 2]
```diff
@@ -397,65 +398,65 @@ msgstr "Duration in h"
...
 
 #. German translation: Der Name der Aufgabe
 #: ./opengever/task/browser/delegate/metadata.py
 #: ./opengever/task/task.py
 msgid "help_title"
-msgstr "help_title"
+msgstr ""
```
Comment (njohner) [inline opengever/task/locales/en/LC_MESSAGES/opengever.task.po:449]: ?

File: opengever/task/locales/en/LC_MESSAGES/opengever.task.po [hunk 3]
```diff
@@ -863,25 +865,25 @@ msgstr "Deadline successfully changed."
...
 msgid "msg_not_revoking_permissions"
-msgstr "Temporary permissions will not be revoked by this transtion as revoke permissions is unchecked for this task"
+msgstr "Temporary permissions for the responsible user (and their deputy) will not be revoked by this transition because that option was disabled for this particular task."
 
 #. German translation: Diese Statusänderung führt dazu, dass temporäre Berechtigungen für den Auftragnehmer und dessen Stellvertretung entzogen werden.
 #. Default: "This transtion revokes temporary permissions for the responsible user and agency group."
 #: ./opengever/task/response.py
 msgid "msg_revoking_permissions"
-msgstr "This transtion revokes temporary permissions for the responsible user and agency group."
+msgstr "Temporary permissions for the responsible user (and their deputy) will be revoked by performing this transition."
```
Comment (njohner) [inline opengever/task/locales/en/LC_MESSAGES/opengever.task.po:874]: be revoked by performing -> be revoked when performing

File: opengever/task/locales/en/LC_MESSAGES/opengever.task.po [hunk 4]
```diff
@@ -987,7 +989,7 @@ msgstr "Response"
 #. Default: "The given deadline, is the current one."
 #: ./opengever/task/browser/modify_deadline.py
 msgid "same_deadline_error"
-msgstr "The given deadline, is the current one."
+msgstr "Given deadline is the same as the current one."
```
Comment (njohner) [inline opengever/task/locales/en/LC_MESSAGES/opengever.task.po:992]: The entered deadline?
The deadline set?

**Review by njohner — APPROVED** [review]
🎉 LGTM, just a few comments

File: opengever/task/locales/en/LC_MESSAGES/opengever.task.po (hunk 1 above)
Comment (lukasgraf) [reply opengever/task/locales/en/LC_MESSAGES/opengever.task.po:443]: I spent a while thinking of a good translation for the German sentence, but then decided to drop those two descriptions alltogether. Even the German one is silly. So either very detailed instructions or just a comment. So basically, put anything you'd like in the text field. I don't think people need encouragement to do that 😉 
But I'll see if I can come up with a phrasing that isn't too bad.

**Pull request merged**